mediaflow_proxy/routes/playlist_builder.py

```
@@ -0,0 +1,388 @@
+        return StreamingResponse(
+            generate_response(),
+            media_type='application/vnd.apple.mpegurl',
+            headers={
+                'Content-Disposition': 'attachment; filename="playlist.m3u"',
+                'Access-Control-Allow-Origin': '*'
+            }
+        )
+        
+    except Exception as e:
+        print(f"ERRORE GENERALE: {str(e)}")
+        raise HTTPException(status_code=500, detail=f"Errore: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)